crypto/ocsp/ocsp_http.c

```diff
@@ -14,19 +14,20 @@
 #ifndef OPENSSL_NO_OCSP
 
 # ifndef OPENSSL_NO_DEPRECATED_3_0
-int OCSP_REQ_CTX_set1_req(OCSP_REQ_CTX *rctx, const OCSP_REQUEST *req)
+int OCSP_REQ_CTX_set1_req(OSSL_HTTP_REQ_CTX *rctx, const OCSP_REQUEST *req)
 {
-    return OCSP_REQ_CTX_i2d(rctx, "application/ocsp-request",
-                            ASN1_ITEM_rptr(OCSP_REQUEST), (ASN1_VALUE *)req);
+    return OSSL_HTTP_REQ_CTX_i2d(rctx, "application/ocsp-request",
+                                 ASN1_ITEM_rptr(OCSP_REQUEST),
+                                 (ASN1_VALUE *)req);
 }
 # endif
 
-OCSP_REQ_CTX *OCSP_sendreq_new(BIO *io, const char *path, OCSP_REQUEST *req,
-                               int maxline)
+OSSL_HTTP_REQ_CTX *OCSP_sendreq_new(BIO *io, const char *path,
```

Comment:
This change in return type is also worthy of mention in the CHANGES entry IMO.

Reply:
The changes entry does say that the type OCSP_REQ_CTX has been replaced with OSSL_HTTP_REQ_CTX, is that not good enough?

Reply:
So it does. Yeah ok.